main.js

```diff
@@ -160,4 +160,135 @@ const inventory = [
     originalStock: 10,
     sold: 8,
   },
-];
\ No newline at end of file
+];
+
+// Opdracht 1a
+
+const tvTypes = inventory.map((tvType) => {
+  return tvType.type;
+})
+
+console.log(tvTypes);
+
+// Opdracht 1b
+
+const soldOutTvs = inventory.filter((soldOutTv) => {
+  return soldOutTv.sold >= soldOutTv.originalStock;
+})
+
+console.log(soldOutTvs);
+
+// Opdracht 1c
+
+const haveAmbilight = inventory.filter((hasAmbilight) => {
+  if (hasAmbilight.options.ambiLight === true) {
+    return true;
+  }
```

Comment:
De filter functie moet eigenlijk altijd een boolean teruggeven. Nu geef je óf `true` of `undefined` terug. Daarnaast zou je dit kunnen versimpelen naar: `return hasAmbilight.options.ambiLight;`